src/pudl/package_data/meta/datapkg/datapackage.json

```diff
@@ -3648,6 +3714,12 @@
                         "description": "If there is an ash impoundment at the plant, the ash impoundment status as of December 31 of the reporting year.",
                         "format": "default"
                     },
+                    {
+                        "name": "coordinate_system",
+                        "type": "string",
+                        "description": "Projection",
+                        "format": "default"
+                    },
```

Comment:
Can we offer a more descriptive description here? Is it a map projection? What naming system is it coming from? Why is a map projection required? Is it meant to be applied to the latitude / longitude values for the plant?

Reply:
Ug, this column only shows up in the 2007 data....I almost want to delete it because it contains so little information. It's also not mentioned at all in the layout file, so there really isn't any further description available other than our own inferences.